### ISSUE TITLE
General protocol for z-ordering of tools?

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/tasks#1009]
1. Does anyone know of a general protocol for the z-ordering of tools like timers, rulers, etc.?
2. Should there be a general protocol for the z-ordering of tools?

See this [issue writeup](https://github.com/phetsims/gas-properties/issues/148#issue-474282628) (not the whole issue) and [this comment](https://github.com/phetsims/gas-properties/issues/148#issuecomment-516569707) for clarification.

From Slack:

Liam Mulhall [4:56 PM]
https://github.com/phetsims/gas-properties/issues/148#issuecomment-516569707
Should this be the protocol for all sims? Or should the z-ordering of tools be determined on a case-by-case basis?

Amy Rouinfar [11:02 PM]
I think that’d generally be my preference, though there may occasionally be a sim-specific exception.

**Comment by amanda-phet** [comment]
I can't really comment because I'd need a bunch of sims to think about for context. For "tools" like meters and readouts, I suppose they should be layered by whichever was last clicked, but there are so many unique cases I don't think we can generalize for all sims...

**Comment by ariel-phet** [comment]
@lmulhall-phet @amanda-phet @arouinfar 

This is a tough one...I do agree that generally having the "active" tool on top seems to makes sense. That should probably be our "default" thought - but I think we are still going to need to determine behavior on a sim-by-sim basis since due to the vast nature of tools we use.

**Comment by arouinfar** [comment]
> This is a tough one...I do agree that generally having the "active" tool on top seems to makes sense. That should probably be our "default" thought

Agreed. I think it could be worthwhile for the "default thought" to be added to the QA handbook. When QA looks at the z-ordering of tools, this should generally be the behavior they're looking for. If the behavior is different, double check with the design doc and/or lead designer.
